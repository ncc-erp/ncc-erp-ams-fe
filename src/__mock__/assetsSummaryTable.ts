export default {
  categories: [
    {
      id: 5,
      name: "PC",
      assets_count: 93,
      status_labels: [
        {
          id: 1,
          name: "Pending",
          assets_count: 0,
        },
        {
          id: 3,
          name: "Broken",
          assets_count: 0,
        },
        {
          id: 4,
          name: "Assign",
          assets_count: 89,
        },
        {
          id: 5,
          name: "Ready to deploy",
          assets_count: 4,
        },
      ],
    },
    {
      id: 6,
      name: "Monitor",
      assets_count: 135,
      status_labels: [
        {
          id: 1,
          name: "Pending",
          assets_count: 0,
        },
        {
          id: 3,
          name: "Broken",
          assets_count: 1,
        },
        {
          id: 4,
          name: "Assign",
          assets_count: 116,
        },
        {
          id: 5,
          name: "Ready to deploy",
          assets_count: 18,
        },
      ],
    },
    {
      id: 7,
      name: "Mouse",
      assets_count: 95,
      status_labels: [
        {
          id: 1,
          name: "Pending",
          assets_count: 0,
        },
        {
          id: 3,
          name: "Broken",
          assets_count: 5,
        },
        {
          id: 4,
          name: "Assign",
          assets_count: 85,
        },
        {
          id: 5,
          name: "Ready to deploy",
          assets_count: 5,
        },
      ],
    },
    {
      id: 8,
      name: "Keyboard",
      assets_count: 88,
      status_labels: [
        {
          id: 1,
          name: "Pending",
          assets_count: 0,
        },
        {
          id: 3,
          name: "Broken",
          assets_count: 4,
        },
        {
          id: 4,
          name: "Assign",
          assets_count: 81,
        },
        {
          id: 5,
          name: "Ready to deploy",
          assets_count: 3,
        },
      ],
    },
    {
      id: 10,
      name: "Headphone",
      assets_count: 58,
      status_labels: [
        {
          id: 1,
          name: "Pending",
          assets_count: 0,
        },
        {
          id: 3,
          name: "Broken",
          assets_count: 4,
        },
        {
          id: 4,
          name: "Assign",
          assets_count: 54,
        },
        {
          id: 5,
          name: "Ready to deploy",
          assets_count: 0,
        },
      ],
    },
    {
      id: 13,
      name: "Device",
      assets_count: 61,
      status_labels: [
        {
          id: 1,
          name: "Pending",
          assets_count: 0,
        },
        {
          id: 3,
          name: "Broken",
          assets_count: 0,
        },
        {
          id: 4,
          name: "Assign",
          assets_count: 61,
        },
        {
          id: 5,
          name: "Ready to deploy",
          assets_count: 0,
        },
      ],
    },
  ],
};
