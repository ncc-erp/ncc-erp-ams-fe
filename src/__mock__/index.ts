export { default as assetsSummaryPieChart } from "./assetsSummaryPieChart";
export { default as assetsSummaryTable } from "./assetsSummaryTable";
