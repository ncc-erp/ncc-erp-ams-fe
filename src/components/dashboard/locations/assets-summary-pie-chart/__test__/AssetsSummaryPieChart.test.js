import renderer from "react-test-renderer";
import { AssetsSummaryPieChart } from "..";
import { assetsSummaryPieChart } from "../../../../../__mock__";

test("Test AssetsSummaryPieChart component", () => {
  const component = renderer.create(
    <AssetsSummaryPieChart {...assetsSummaryPieChart}></AssetsSummaryPieChart>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
