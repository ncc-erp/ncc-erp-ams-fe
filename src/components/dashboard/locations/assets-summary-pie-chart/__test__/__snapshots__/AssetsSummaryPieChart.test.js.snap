// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test AssetsSummaryPieChart component 1`] = `
<div
  style={
    Object {
      "height": "inherit",
    }
  }
/>
`;
